core/src/main/java/org/frankframework/management/bus/MessageDispatcher.java

```diff
@@ -116,7 +117,7 @@ private void findServiceActivators(BeanDefinition beanDef) throws ClassNotFoundE
 	}
 
 	private void registerServiceActivator(Object bean, Method method, SubscribableChannel channel, BusTopic topic) {
-		String componentName = bean.getClass().getSimpleName()+"."+method.getName();
+		String componentName = ClassUtils.classNameOf(bean)+"."+method.getName();
```

Comment:
Fixes `... ServiceActivator [GetSchedules$$EnhancerBySpringCGLIB$$deff747.getSchedules] on ...`
By removing the EnhancerBySpringCGLIB